src/pages/guides/getting_started/code_playground_workflow.md

```diff
@@ -0,0 +1,193 @@
+
+## Transitioning Between Modes
+
+### From Script Mode to Add-on Mode
+
+Once you've tested your code in Script mode, you can easily transition it into Add-on Mode to build a user interface around your functionality:
+
+1. **Copy Your Code**: Use the **Copy** button in the right corner to quickly copy your code to the clipboard
+2. **Switch to Add-on Mode**: Click the **Add-on** button to enter Add-on Mode
+3. **Paste and Adapt**: Paste the code into the **Document JS** tab. **Note:** Don't forget you'll need to add the `import` statements for the Document APIs and handle your `async` functions manually in this mode
+4. **Build Your UI**: Modify your script code to be used in the add-on context along with your front-end logic in the **HTML**, **Iframe JS**, and **CSS** tabs. Use the initial sample code provided as a reference
+5. **Configure Manifest**: If you set any manifest properties (e.g., **experimentalApis**) while in Script mode, make sure to set the same in Add-on Mode as well. These settings only apply to the context of the development mode you're in
+6. **Test Your Add-on**: Click the **Run Code** button to execute your code within the context of your add-on
+
+## Keyboard Shortcuts
+
+Use these keyboard shortcuts to work more efficiently:
+
+| Action                           | Windows/Linux                                                | macOS                                                       |
+| -------------------------------- | ------------------------------------------------------------ | ----------------------------------------------------------- |
+| **Save**                         | <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>S</kbd>            | <kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>S</kbd>            |
+| **Run**                          | <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>Return/Enter</kbd> | <kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>Return/Enter</kbd> |
+| **Reset**                        | <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>X</kbd>            | <kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>X</kbd>            |
+| **Increase font size**           | <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>Plus (+)</kbd>     | <kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>Plus (+)</kbd>     |
+| **Decrease font size**           | <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>Minus (-)</kbd>    | <kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>Minus (-)</kbd>    |
+| **Switch between tabs**          | <kbd>Ctrl</kbd> + <kbd>1, 2, 3, 4</kbd>                      | <kbd>Cmd</kbd> + <kbd>1, 2, 3, 4</kbd>                      |
+| **View the typings suggestions** | <kbd>Ctrl</kbd> + <kbd>space</kbd>                           | <kbd>Cmd</kbd> + <kbd>space</kbd>                           |
+
+<InlineAlert slots="header,text1" variant="success"/>
+
+#### TIP
+
+Use the "**...**" button in the top right corner of the playground window to reference the available keyboard shortcuts, start a new session, link to documentation and more.
```

Comment:
The UI for code playground has slightly changed. Its now a 'More' button on the top bar.